support/Pipelines/Bbh/Ringdown.yaml

```diff
@@ -144,6 +144,15 @@ EventsAndTriggers:
           - Name: PointwiseL2Norm(FourIndexConstraint)
             NormType: L2Norm
             Components: Sum
+  - Trigger:
+      Slabs:
+        EvenlySpaced:
+          Interval: 10
+          Offset: 0
+    Events:
+      - ErrorIfDataTooBig:
+          Threshold: 100
+          VariablesToCheck: [SpacetimeMetric]
```

Comment:
Question: why not monitor the constraints during the ringdown? No need to hold up the PR aboutt this, I'm just surprised

Reply:
I don't actually know, but if I were to venture a guess it'd be because the constraint violations at t=Ringdown may be pretty large initially and then settle down quickly, so that initial violation could have been over the threshold? But then why not just delay the trigger? I'm not really sure. We should ask @markscheel at the BBH meeting today.

Reply:
Sounds good! I think SpEC actually monitors the constraints _outside_ the horizons, but that's obviously harder to do.